composer.json

```diff
@@ -10,9 +10,12 @@
     "license": "MIT",
     "authors": [
         {
-            "name": "Will Pillar",
-            "email": "will.pillar@graze.com"
+            "name": "Graze Developers",
+            "email": "developers@graze.com"
         }
     ],
-    "require": {}
+    "require": {},
+    "require-dev": {
+        "squizlabs/php_codesniffer": "^3.0"
```

Comment:
What was the reason behind including this? It doesn't seem to be used.

There is some irony in the fact that the code in this repo doesn't conform to our code standards. 😆  I think that should be fixed but that can probably be left to another branch.